server/src/main/java/org/elasticsearch/cluster/metadata/MetadataIndexTemplateService.java

```diff
@@ -116,6 +118,58 @@ public class MetadataIndexTemplateService {
                     .map(defaultTimestampField)
                     .endObject()
             );
+            DATA_STREAM_FAILURE_STORE_MAPPING = new CompressedXContent(
+                (builder, params) -> builder.startObject(MapperService.SINGLE_MAPPING_NAME)
```

Comment:
Do you think we should add a comment with the JSON equivalent to this, just so someone looking at the code trying to parse this back into JSON can see what it ends up looking like?

Reply:
Maybe store this as an actual JSON file? This would a void inconsistencies between the comment and the code.